.github/workflows/compat-docker-latest.yml

```diff
@@ -1,20 +1,20 @@
-name: Compat Check Docker - Lightning@latest
+name: Compat Check Docker - Lightning@stable
```

Comment:
Stable because, our latest already diverged from stable, right?

Reply:
Should we rename the script to stable? Or do we use it for latest too?

Reply:
I realized that we refer to the latest official release as "stable" and the development (also our master branch) as "latest". I'm thus adopting the convention in the Docker workflow.

Reply:
In this case, should we rename the script to:
.github/workflows/compat-docker-stable.yml ?

Reply:
Good point. Will do.